gramps/gen/proxy/proxybase.py

```diff
@@ -308,83 +348,88 @@ def get_tag_handles(self, sort_handles=False, locale=glocale):
         else:
             return []
 
-    def get_default_person(self):
+    def get_default_person(self) -> Person | None:
         """returns the default Person of the database"""
         return self.db.get_default_person()
 
-    def get_default_handle(self):
+    def get_default_handle(self) -> PersonHandle | None:
         """returns the default Person of the database"""
         return self.db.get_default_handle()
 
-    def iter_person_handles(self):
+    def iter_person_handles(self) -> Generator[PersonHandle]:
         """
         Return an iterator over database handles, one handle for each Person in
         the database.
         """
-        return filter(self.include_person, self.db.iter_person_handles())
+        return (_ for _ in filter(self.include_person, self.db.iter_person_handles()))
```

Comment:
```
        return (_ for _ in filter(self.include_person, self.db.iter_person_handles()))
```
Flagging this change for comment.
Method: `def iter_person_handles(self) -> Generator[PersonHandle]`

This code previously returned a List. The equivalent method in `DbGeneric` returns a `Generator[PersonHandle]`.
I've standardised on returning a generator. Is this the right choice?

Reply:
Sounds good to me.

Reply:
`collections.Iterable` is the common denominator between `list` and `Generator`.